.github/workflows/package.yml

```diff
@@ -2,99 +2,79 @@ name: Package
 
 on:
   push:
-    branches: [ master ]
+    branches: [master]
 
 env:
   CARGO_TERM_COLOR: always
   cargo-deb-version: 1.28.0
 
 jobs:
-  build-armv7:
+  build:
     runs-on: ubuntu-latest
-    env:
-      target: armv7-unknown-linux-gnueabihf
-      binutils: binutils-arm-linux-gnueabihf
+    strategy:
+      matrix:
+        target:
+          - armv7-unknown-linux-gnueabihf
+          - aarch64-unknown-linux-gnu
+          - # Empty target to build for host architecture (x86_64-unknown-linux-gnu) without cross.
```

Comment:
Can we put the target as x86_64-unknown-linux-gnu here to make it slightly less magical? (even if we *don't* decide use cross for this target)

Reply:
If we can make `cross` work then sure, but if not then I think matching on this target string rather than just an empty string to not use cross is worse.